doc/src/install.rst

```diff
@@ -44,7 +44,7 @@ To update to the latest version, go into your repository and execute::
 If you want to install SymPy, but still want to use the git version, you can run
 from your repository::
 
-    setupegg.py develop
+    python setupegg.py develop
```

Comment:
Why not `setup.py`?

Reply:
SymPy setup.py is pure distutils. setupegg.py is setuptools.

Reply:
@moorepants `setup.py` tries to import `setuptools` in:
https://github.com/sympy/sympy/blob/5fcbde2ed7385df250510bd5207f270cba990265/setup.py#L44-L51

Reply:
I'm not remembering exactly why setupegg.py is needed. I remember convincing myself years ago that it was useful. Maybe it was due to limitations of past versions of setuptools or something.

Reply:
Oh, I see why this is. If you do `python setup.py develop` you'll get an error related to a missing command if you don't have setuptools installed and are only utilizing distutils. But if you do `python setupegg.py develop` you'll get an import error instead. So maybe we should just get rid of setupegg.py and fix the error handling in setup.py so that the error is "Install setuptools" if you do `python setup.py develop` and only have distutils.